plone/app/registry/exportimport/handler.py

```diff
@@ -21,6 +21,36 @@
 _marker = object()
 
 
+def evaluateCondition(expression):
+    """Evaluate import condition.
+
+    ``expression`` is a string of the form "verb arguments".
+
+    Currently the supported verbs are ``installed`` and ``not-installed``.
+    """
+    arguments = expression.split(None)
+    verb = arguments.pop(0)
+
+    if verb in ('installed', 'not-installed'):
+        if not arguments:
+            raise ValueError("Package name missing: %r" % expression)
+        if len(arguments) > 1:
+            raise ValueError("Only one package allowed: %r" % expression)
+
+        try:
+            __import__(arguments[0])
+            installed = True
+        except ImportError:
+            installed = False
+
+        if verb == 'installed':
+            return installed
+        elif verb == 'not-installed':
+            return not installed
+    else:
+        raise ValueError("Invalid import condition: %r" % expression)
```

Comment:
@datakurre just a nice to have that I tend to do lately: reverse the if conditional so you can error early and then the big if block can be dedented on level and makes code clearer